ocs_ci/deployment/aws.py

```diff
@@ -79,6 +79,18 @@ def add_node(self):
         # TODO: Implement later
         super(AWSBase, self).add_node()
 
+    def check_cluster_existence(self):
+        """
+        Check cluster existence
+
+        """
+        cluster_name = config.ENV_DATA['cluster_name']
+        pattern = cluster_name.split("-")[0] + '*'
```

Comment:
all the jenkins jobs have cluster name similar to: jenkins-us-tier1 , so this is problematic for jenkins executions.

So for this purpose I would allow us to add to the configuration under DEPLOYMENT section some parameter like force_deploy_multiple_clusters or something like this and if it's set to True we will ignore this check.